data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
@WibbenZ what i meant, a user that has a custom datapack with according to lua api version (interfaces and methods), its datapack may work without any imposed script by server, such this one